src/plfa/part2/Confluence.lagda.md

```diff
@@ -605,19 +605,18 @@ Broadly speaking, this proof of confluence, based on parallel
 reduction, is due to W. Tait and P. Martin-Lof (see Barendredgt 1984,
 Section 3.2).  Details of the mechanization come from several sources.
 The `subst-par` lemma is the "strong substitutivity" lemma of Shafer,
-Tebbi, and Smolka (ITP 2015). The proofs of `par-diamond`, `strip`,
-and `par-confluence` are based on Pfenning's 1992 technical report
-about the Church-Rosser theorem. In addition, we consulted Nipkow and
+Tebbi, and Smolka (ITP 2015). The proofs of `par-triangle`, `strip`,
+and `par-confluence` are based on the notion of complete development
+by Takahashi (1995) and Pfenning's 1992 technical report about the
+Church-Rosser theorem. In addition, we consulted Nipkow and
 Berghofer's mechanization in Isabelle, which is based on an earlier
-article by Nipkow (JAR 1996).  We opted not to use the "complete
-developments" approach of Takahashi (1995) because we felt that the
-proof was simple enough based solely on parallel reduction.  There are
-many more mechanizations of the Church-Rosser theorem that we have not
-yet had the time to read, including Shankar's (J. ACM 1988) and
-Homeier's (TPHOLs 2001).
+article by Nipkow (JAR 1996).  There are many more mechanizations of
+the Church-Rosser theorem that we have not yet had the time to read,
```

Comment:
Not sure if this admission is warranted in a textbook?

Reply:
Removed.